src/components/Inputs/InputRow.tsx

```diff
@@ -0,0 +1,18 @@
+import cn from 'classnames';
+
+interface Props extends React.ComponentPropsWithoutRef<'input'> {
+  label?: string;
+}
+
+export const InputRow = ({ className, id, label, type, ...rest }: Props): React.ReactElement => {
+  return (
+    <div className={cn('flex w-full flex-col gap-1', className)}>
+      {label && (
+        <label htmlFor={id} className="font-poly italic text-[#A2AAB6]">
+          {label}
+        </label>
+      )}
+      <input type={type ?? 'text'} className="border-b-2 py-1" {...rest}></input>
```

Comment:
need the input text to be #333F4C - 
![image](https://user-images.githubusercontent.com/88429620/217223278-cd54a94d-7618-4ad3-832f-1a0116c5b680.png)